src/main/java/com/maxtrain/prsspringboot/controllers/UserController.java

```diff
@@ -85,10 +87,15 @@ public User delete(@PathVariable int id) {
 	
 		
 	@PostMapping("/login")
-	public User authenticate(@RequestBody User loginUser) {
-		User user = new User();
-		user = userRepo.findByUserNameAndPassword(loginUser.getUserName(), loginUser.getPassword());
-		
-		return user;
+	public AuthenticationResponse authenticate(@RequestBody AuthenticateRequest loginUser) {
```

Comment:
I don't like the use of AuthenticationResponse vs. authenticatRequest...please change the response object to AuthenticateResponse